typography-nutrition-label/index49.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+    <link href="./css/style49.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="label">
+      <header>
+        <h1 class="bold">Nutrition Facts</h1>
+        <div class="divider"></div>
+        <p>8 servings per container</p>
+        <p class="bold">Serving size <span class="right">2/3 cup (55g)</span></p>
+      </header>
+      <div class="divider lg"></div>
+      <div class="calories-info">
+        <p class="bold sm-text">Amount per serving</p>
+        <h1>Calories <span class="right">230</span></h1>
+      </div>
+      <div class="divider md"></div>
+      <div class="daily-value sm-text">
+        <p class="right bold">% Daily Value *</p>
+        <!--
+        Because the height of floated elements don't add to the height of their container, the `<p>` element you just created is outside of the label's border.
```

Comment:
I wonder if this text could be simplified. Maybe something like: "The `float: right` styling couases the `<p>` element to be outside of the label's border. Clear the float by adding a divider after the last `<p>` element."

Reply:
Yes, I agree this could be much simpler. Thank you for your help with the rewrite! I'll make this change now.